motion/p5js/cones/index.js

```diff
@@ -0,0 +1,72 @@
+const height = 60;
+const radius = 60;
+let y = 0;
+let increment = 1;
+
+function setup() {
+  createCanvas(500, 500, WEBGL);
+  frameRate(30);
+}
+
+function drawCones(count, current) {
+  current = current || 1;
+
+  if (current > count) {
+    return;
+  }
+
+  cone(radius, height);
+
+  push();
+
+  translate(2 * radius, 0);
+  drawCones(count, current + 1);
+
+  pop();
+}
+
+function drawRotated(count, position) {
+  push();
+
+  rotateX(radians(90));
+  drawCones(count, position);
+
+  pop();
+}
+
+function draw() {
+  camera(0, 0, 300);
+  background(250);
+
+  y = y + increment;
+
+  if (y === 0) {
+    increment = 1;
+  } else if (y === height) {
+    increment = -1;
+  }
+}
+
+function draw(){
+  pointLight(250, 250, 250, y * 8, y * 8, 0);
+
+  translate(0, -4 * radius);
+
+  drawRotated(1);
+
+  translate(-2 * radius, 2 * radius);
+
+  drawRotated(3);
+
+  translate(-2 * radius, 2 * radius);
+
+  drawRotated(5);
+
+  translate(2* radius, 2 * radius);
+
+  drawRotated(3);
+
+  translate(2 * radius, 2 * radius);
+
+  drawRotated(1);
+}
```

Comment:
Hey, I help @natascher from time to time with her assignments and I don't get some of the code comments. Why are those white-spaces unnecessary? They separate the function which positions the canvas from the function that draws on the canvas. This way parts that logically belong together (in this case always one line) is surrounded by white-space. Code optimisations such das removing white-space should be done by a built system, not the developer.